src/components/RealTimeInterface/SessionImageCapture.vue

```diff
@@ -7,13 +7,13 @@ const status = ref(null)
 let pollingInterval = null
 const statusApiUrl = 'http://rti-bridge-dev.lco.gtn/status'
```

Comment:
can we call this telescopeStatusApiUrl? also this API url should live in some global config, rather than being hardcoded.

Reply:
yes, it is in a global config and in the section that wasn't relevant to my beloved code reviewers, I mentioned that this logic will be implemented soon just not yet. Not sure if you remember when I was trying to set up the configuration store where it wasn't working. So def something I have to look into and that's why some of the code is commented out. In the meantime, I'll rename this const : )